app/controllers/AnnotationIOController.scala

```diff
@@ -320,7 +371,7 @@ Expects:
         _ = fetchedVolumeLayers.zipWithIndex.map {
           case (volumeLayer, index) =>
             volumeLayer.volumeDataOpt.foreach { volumeData =>
-              val dataZipName = volumeLayer.volumeDataZipName(index, fetchedSkeletonLayers.length == 1)
+              val dataZipName = volumeLayer.volumeDataZipName(index, fetchedVolumeLayers.length == 1)
```

Comment:
This was a bug: the data.zips should get their index in the file name if there are multiple *volume* layers, not *skeleton* layers